holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1270 @@
+
+    dirpath_ = dirpath.rstrip('/') + '/'
+
+    logger = logger or pdslogger.PdsLogger.get_logger(LOGNAME)
+    logger.replace_root(pdsdir.root_)
+
+    merged_limits = LOAD_LINKS_LIMITS.copy()
+    merged_limits.update(limits)
+    logger.open('Reading link shelf file for', dirpath, limits=merged_limits)
+
+    try:
+        (link_path, lskip) = pdsdir.shelf_path_and_lskip('link')
```

Ruff (0.14.5)

Unpacked variable `lskip` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)